### DISCUSSION TITLE
How to enable tty support

### DISCUSSION THREAD

**Discussion by acrobat** [community/community#26463] [Actions]
<p>We receive an error during our test suite that /dev/tty is not writeable. </p>
<pre><code>TTY mode requires /dev/tty to be read/writable.
</code></pre>
<p>Is there a way to enable this in github actions or to workaround this problem?</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/acrobat">@acrobat</a> ,</p>
<p>Please try <strong>set(‘git_tty’, false);</strong> to see if the problem can be solved, like mentioned in <a href="https://github.com/deployphp/deployer/issues/1713" rel="nofollow noopener">this ticket</a> for the similar problem.</p>
